app/models/reports/repository.rb

```diff
@@ -179,15 +179,38 @@ def initialize(regions, periods:)
       end
     end
 
-    # We determine this rate via subtraction from 100 instead of via division to avoid confusing rounding errors.
-    memoize def missed_visits_rate
+    memoize def missed_visits_rate_without_ltfu
       region_period_cached_query(__method__) do |entry|
         slug, period = entry.slug, entry.period
-        remaining_percentages = controlled_patients_rate[slug][period] + uncontrolled_patients_rate[slug][period] + visited_without_bp_taken_rate[slug][period]
-        100 - remaining_percentages
+        visit_rates = controlled_patients_rate[slug][period] + uncontrolled_patients_rate[slug][period] + visited_without_bp_taken_rate[slug][period]
+        100 - visit_rates
       end
     end
 
+    def missed_visits_with_ltfu
+      region_period_cached_query(__method__, with_ltfu: true) do |entry|
+        slug = entry.slug
+        patient_count = denominator(entry.region, entry.period, with_ltfu: true)
+        controlled = controlled_patients_count[slug][entry.period]
+        uncontrolled = uncontrolled_patients_count[slug][entry.period]
+        visits = visited_without_bp_taken[slug][entry.period]
+        patient_count - visits - controlled - uncontrolled
+      end
+    end
+
+    memoize def missed_visits_rate_with_ltfu
+      region_period_cached_query(__method__, with_ltfu: true) do |entry|
+        slug, period = entry.slug, entry.period
+        visit_rates = controlled_patients_rate(with_ltfu: true)[slug][period] +
+          uncontrolled_patients_rate(with_ltfu: true)[slug][period] +
+          visited_without_bp_taken_rate(with_ltfu: true)[slug][period]
+        100 - visit_rates
+      end
+    end
+
+    alias_method :missed_visits, :missed_visits_without_ltfu
+    alias_method :missed_visits_rate, :missed_visits_rate_without_ltfu
```

Comment:
The default behavior for missed visits is `without_ltfu`, so we setup aliases.  Maybe we'll remove them in the future for more explicit names here.